packages/tinybird/src/validation.ts

```diff
@@ -69,7 +69,8 @@ export const tbParameterResponseList = z.object({
   monitorId: z.string().default(""), // REMINDER: remove default once alpha
   fromDate: z.number().int().default(0), // always start from a date
   toDate: z.number().int().optional(),
-  limit: z.number().int().optional().default(2500), // one day has 2448 pings (17 (regions) * 6 (per hour) * 24)
+  page: z.number().int().optional().default(0),
+  page_size: z.number().int().optional().default(20), // one day has 2448 pings (17 (regions) * 6 (per hour) * 24)
```

Comment:
Is it possible to keep `limit` and call the other parameter `offset` instead of `page`? That way, we stay close to the SQL syntax.

Reply:
@mxkaske we sure can do that. It won't change much.

Reply:
@mxkaske so instead of page and page_size, should we limit and offset ??

Reply:
@mxkaske done